charts/kube-prometheus-stack/values.yaml

```diff
@@ -1982,6 +2006,10 @@ kubeScheduler:
     additionalLabels: {}
     #  foo: bar
 
+    ## defines the labels which are transferred from the associated Kubernetes Service object onto the ingested metrics.
+    ## https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#servicemonitor
+    targetLabels: []
```

Comment:
It would be make sense to allow end-users to define additional labels on that service? Otherwise, it might be pointless.

https://github.com/prometheus-community/helm-charts/blob/f828f547380ce33ea5bd7d7b483bb573a8df1a70/charts/kube-prometheus-stack/templates/exporters/kube-etcd/service.yaml#L6-L9

A value like `Values.kubeEtcd.service.labels`

Reply:
As per kube-prometheus-stack.labels function we can supply additional labels to services through .Values.commonLabels which will be passed to targetLabels (once they match) using serviceMonitor.

Reply:
So, basically instead of adding relabeling for each we just add serviceMonitor targetLabels to fetch the respective service label if they match. Would be easy to maintain overrides for multiple clusters.